src/core_plugins/table_vis/public/table_vis_params.html

```diff
@@ -1,6 +1,6 @@
 <div class="form-group">
-  <label>Per Page</label>
-  <input type="number" ng-model="vis.params.perPage" class="form-control">
+  <label id="datatableVisualizationPerPage">Per Page</label>
+  <input type="number" ng-model="vis.params.perPage" class="form-control" for="datatableVisualizationPerPage">
```

Comment:
`id` and `for` are interchanged. The `label` is `for` a specific `input` (specified by its `id`).